tests/integration/helpers/cluster.py

```diff
@@ -378,7 +378,6 @@ def __init__(
         self.env_variables["ASAN_OPTIONS"] = "use_sigaltstack=0"
         self.env_variables["CLICKHOUSE_WATCHDOG_ENABLE"] = "0"
         self.env_variables["CLICKHOUSE_NATS_TLS_SECURE"] = "0"
-        self.env_variables["SPARK_PATH"] = os.environ.get("SPARK_HOME")
```

Comment:
This line can be removed and it should work without it, I forgot to remove it.
So I think you can just delete the line 381 and that's all, no need to move.